src/petals/server/handler.py

```diff
@@ -378,6 +380,23 @@ def _log_request(
         else:
             logger.warning(f"{message}: {warning}")
 
+    async def rpc_info(self, request: runtime_pb2.ExpertUID, context: P2PContext) -> runtime_pb2.ExpertInfo:
+        """Return metadata about stored block uids and current load"""
+        rpc_info = {}
+        if request.uid:
+            backend = self.module_backends[request.uid]
+            rpc_info.update(self.module_backends[request.uid].get_info())
+        else:
+            backend = next(iter(self.module_backends.values()))
```

Comment:
```suggestion
            backend = next(iter(self.module_backends.values()))
            # not saving keys to rpc_info since user did not request any uid
```